.circleci/config.yml

```diff
@@ -148,4 +202,14 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - igor/nightly-build
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - igor/nightly-build
```

Comment:
Should be changed to `master` when merged? Or could we run all branches?

Reply:
Yes, I was thinking in changing it to `master`.
If we program all branches to run at the same time it can generate a large queue.
I haven't found a filter to run only active branches.

Reply:
https://circleci.com/docs/2.0/configuration-reference/#branches-1 seems to imply that only pushed commits schedule to a build. Could you verify that experimentally?

Reply:
Yeah, the default behavior is to trigger the workflow when pushing to a branch. With the keyword `schedule`  it behaves a bit different.
I will schedule two consecutive builds (to avoid waiting two days) without committing between them to verify if in that case a commit is needed.

Reply:
It runs the workflow on schedule regardless of having no new commit.
See for example (hourly builds): https://circleci.com/gh/IgorBaratta/dolfinx

Reply:
Too bad, good digging. Ok, do it the way you think is the best.